src/components/ChatBox.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+interface ChatBoxProps {
+  message: string;
+  width?: number;
+  height?: number;
+  animationDuration?: number; 
+  variant?: 'default' | 'flip'; // Added variant prop
+}
+
+const ChatBox: React.FC<ChatBoxProps> = ({ 
+  message, 
+  width = 700, // Updated default width to 500
```

Comment:
You mentioned // Updated default width to 500 in the comment, but the actual default is 700. If you intended for it to be 500, update the code or remove the comment to avoid confusion.